iele.md

```diff
@@ -1419,7 +1420,7 @@ For each `call*` operation, we make a corresponding call to `#call` and a state-
 
     syntax KItem ::= "#codeDeposit" Int Int Contract LValue LValue Bool
                    | "#mkCodeDeposit" Int Int Contract LValue LValue Bool
-                   | "#finishCodeDeposit" Int Contract LValue LValue Bool
+                   | "#finishCodeDeposit" Int LValue LValue Bool
```

Comment:
It looks to me like none of #codeDeposit, #mkCodeDeposit need the CODE parameter either actually.

Reply:
Sounds right, I have removed it from those too.